src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,70 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "cinema_hall_id")
```

Comment:
```suggestion
    private Long id;
    @ManyToOne
    @JoinColumn(name = "movie_id")
    private Movie movie;
    @ManyToOne
    @JoinColumn(name = "cinema_hall_id")
```
by default `ManyToOne` and `OneToOne` have `FetchType.EAGER`

Reply:
I knew about this but left it to see it directly